ClangSharp/ClangSharp.csproj

```diff
@@ -3,6 +3,19 @@
   <PropertyGroup>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <TargetFramework>netstandard2.0</TargetFramework>
+    <Description>ClangSharp are strongly-typed safe Clang bindings written in C# for .NET and Mono, tested on Linux and Windows. ClangSharp is self-hosted and ClangSharp auto-generates itself parsing LLVM-C header files.</Description>
+    <Version>3.9.0</Version>

```

Comment:
@mjsabby, is this meant to be the same version as the `libclang` supported or is it independent?